R/plot-helpers.R

```diff
@@ -59,6 +59,7 @@ create_plot_base <- function(plot_background = NULL) {
 #' @return A \code{ggplot2} instance with the feature added to it
 #'
 #' @keywords internal
+#' @importFrom rlang .data
```

Comment:
@mrcaseb just added this (and subsequent `.data${{variable}}` below) to avoid notes. Otherwise looks good!